apps/ledger-live-mobile/src/families/tron/ConfirmationPostAlert.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Account } from "@ledgerhq/types-live";
+import React from "react";
+import { Trans } from "react-i18next";
+import Alert from "../../components/Alert";
+import { urls } from "../../config/urls";
+
+type Props = {
+  mainAccount: Account;
+};
+
+export default function ReceiveConfirmationAlertPost({ mainAccount }: Props) {
```

Comment:
Sorry to be a pain with the names :smile: but should be `ReceiveConfirmationPostAlert`, otherwise we won't find it when searching the code. All the default exports already don't help with that...

Reply:
Should I do the same for desktop ?